pandas/core/arrays/period.py

```diff
@@ -67,7 +67,7 @@ class PeriodArray(PeriodMixin, dtl.DatelikeOps):
     """
     Pandas ExtensionArray for storing Period data.
 
-    Users should use :func:`period_array` to create new instances.
+    Users should use :func:`period_range` to create new instances.
```

Comment:
period_array is correct

Reply:
It's not public, see also the linked issue